packages/react-generator/src/parser/jsx-slot.js

```
@@ -0,0 +1,118 @@
+      result.push(subTemplate)
+    } else if (children?.type === 'JSExpression') {
+      result.push(`{ ${children.value.replace(/this\./g, '')} }`)
+    } else if (children?.type === 'i18n') {
+      result.push(`{t('${children.key}')}`)
+    } else {
+      result.push(children || '')
+    }
+
+    result.push(`</${component}>`)
+  }
+  condition && result.push(' }')
```

ESLint

[error] Expected an assignment or function call and instead saw an expression.

(@typescript-eslint/no-unused-expressions)